modules/age.nix

```
@@ -14,6 +14,11 @@ with lib; let
 
   users = config.users.users;
 
+  sysusersEnabled =
+    if isDarwin
+    then false
+    else options.systemd ? sysusers;
```

This doesn't seem right. "If the `systemd.sysusers` option exists, then sysuers is enabled"? I would expect this to be something like "If the `systemd.sysusers.enable` config is set to `true`, then sysusers is enabled":
```suggestion
    else config.systemd.sysusers.enable or false;
```

the `or false` should short-circuit in the case that the `systemd.sysusers.enable` option (or any parent option) doesn't exist